sdk/sdk-view/src/test/java/mil/emp3/core/StorageManagerTest.java

```diff
@@ -0,0 +1,462 @@
+package mil.emp3.core;
```

Comment:
I think alot of it would have been redundant.

Testing the Storage/Event manager covers the important bits that contains the logic for passing the userContext object around.

If I were to unit test the ```void addFeatures(List<IFeature> features, boolean visible, Object object)``` method of Overlay, for example, wouldn't we want to just verify the following?:
* if features is null, an EMP_Exception is raised
* if features.size() > 0, the ```validate()``` method was called on each entry
* if features.size() > 0, the ```StorageManager.addFeatures(...)``` method was called with specific arguments

I felt broadening the scope of what was tested would be more akin to functional/integration testing, which would involve an approach similar to what I think you are asking about, and be placed into a different set of code.

Do we need another ticket for that effort?